tests/Codeception/api/com_media/MediaCest.php

```diff
@@ -296,15 +296,64 @@ public function testCreateFolder(ApiTester $I)
 	}
 
 	/**
-	 * Test the PATCH media files endpoint of com_media from the API.
+	 * Test the POST media files endpoint of com_media from the API with adapter information.
 	 *
 	 * @param   mixed   ApiTester  $I  Api tester
 	 *
 	 * @return  void
 	 *
 	 * @since   __DEPLOY_VERSION__
 	 */
-	public function testUpdateFile(ApiTester $I)
+	public function testCreateFileWithAdapter(ApiTester $I)
+	{
+		$I->amBearerAuthenticated($I->getBearerToken());
+		$I->haveHttpHeader('Content-Type', 'application/json');
+		$I->haveHttpHeader('Accept', 'application/vnd.api+json');
+		$I->sendPost(
+			'/media/files',
+			[
+				'path'    => 'local-images:/' . $this->testDirectory . '/test.jpg',
+				'content' => base64_encode(file_get_contents(codecept_data_dir() . '/com_media/test-image-1.jpg'))
+			]
+		);
+
+		$I->seeResponseCodeIs(HttpCode::OK);
+		$I->seeResponseContainsJson(['data' => ['attributes' => ['type' => 'file', 'name' => 'test.jpg']]]);
+	}
+
+	/**
+	 * Test the POST media files endpoint of com_media from the API with adapter information.
+	 *
+	 * @param   mixed   ApiTester  $I  Api tester
```

Comment:
its probably not mixed is it... its `ApiTester` really... 

```suggestion
	 * @param   ApiTester   ApiTester  $I  Api tester
```